contrib/devtools/test_utxo_snapshots.sh

```diff
@@ -64,10 +64,10 @@ trap finish EXIT
 EARLY_IBD_FLAGS="-maxtipage=9223372036854775207 -minimumchainwork=0x00"
 
 server_rpc() {
-  ./src/bitcoin-cli -rpcport=$SERVER_RPC_PORT -datadir="$SERVER_DATADIR" "$@"
+  ./build/src/bitcoin-cli -rpcport=$SERVER_RPC_PORT -datadir="$SERVER_DATADIR" "$@"
```

Comment:
It's not clear to me that hard-coding the build dir into all of these scripts is the right thing to do? Especially if people are likely to be using multiple different dirs, or compare output between builds etc?

Reply:
Thanks, reverted the devtools related changes.

Reply:
> Especially if people are likely to be using multiple different dirs, or want to compare output between builds etc? It seems like a better approach would be to default to `build`, but still let the user specify a dir

I agree with @fanquake, but until #29553 (which will remove this script entirely) gets merged (I'm testing it) and released, the script would fail (same if it get changed and the user specify another dir ok but default is `build` everywhere)...